content/en/guides/get-started/commands.md

```diff
@@ -0,0 +1,316 @@
+
+  </code-block>
+</code-group>
+
+<base-alert type="info">
+
+we recommend putting `.nuxt` in `.npmignore` or `.gitignore`.
+
+</base-alert>
+
+### Static Generated Deployment (Pre-rendered)
+
+Nuxt.js gives you the ability to host your web application on any static hosting.
+
+To deploy a static generated site make sure you have `target: static` in your `nuxt.config.js`.(For Nuxt >= 2.13:)
+
+```js{}[nuxt.config.js]
+export default {
+  target: 'static'
+}
+```
+
+<code-group>
+  <code-block label="npx" active>
+
+```bash
+nuxt generate
+```
+
+  </code-block>
+  <code-block label="Yarn">
+
+```bash
+yarn generate
+```
+
+  </code-block>
+  <code-block label="NPM">
+
+```bash
+npm run generate
+```
+
+  </code-block>
+</code-group>
+
+Nuxt.js will create a `dist` folder with everything inside ready to be deployed on a static hosting service.
+
+#### **Fail on Error**
+
+To return a non-zero status code when a page error is encountered and let the CI/CD fail the deployment or build, you can use the `--fail-on-error` argument.
+
+<code-group>
+<code-block label="npx" active>
+
+```bash
+nuxt generate --fail-on-error
+```
+
+  </code-block>
+  <code-block label="Yarn">
+
+```bash
+yarn generate --fail-on-error
+```
+
+  </code-block>
+  <code-block label="NPM">
+
+```bash
+npm run generate --fail-on-error
+```
+
+  </code-block>
+
+</code-group>
+
+As of Nuxt v2.13 there is a crawler installed that will now crawl your link tags and generate your routes when using the command `nuxt generate` based on those links.
+
+<base-alert>
+
+**Warning:** dynamic routes are ignored by the `generate` command when using Nuxt <= v2.12: [API Configuration generate](/api/configuration-generate#routes)
+
+</base-alert>
+
+<base-alert type="info">
+
+When generating your web application with `nuxt generate`, [the context](/guides/internals-glossary/context) given to [asyncData](/guide/async-data) and [fetch](/guide/vuex-store#the-fetch-method) will not have `req` and `res`.
+
+</base-alert>
+
+### Single Page Application Deployment (SPA)
+
+When ssr is set to false in our `nuxt.config.js` file (`ssr: false`) and we run the `build`/`generate` command, Nuxt will generate our dist folder with all generated routes. These html files common meta and resource links, but not page content. The content is generated in the browser, on the client side.
+
+That means, we can't use SPA'S for applications where content depends on user authentication or a real time API (at least for the first load).
+
+#### **For Static Hosting**
+
+```js{}[nuxt.config.js]
+export default {
+  target: 'static'
+  ssr: false
+}
+```
+
+<code-group>
+  <code-block label="npx" active>
+
+```bash
+nuxt generate
+```
+
+  </code-block>
+  <code-block label="Yarn">
+
+```bash
+yarn generate
+```
+
+  </code-block>
+  <code-block label="NPM">
+
+```bash
+npm run generate
+```
+
+  </code-block>
+</code-group>
+
+Deploy the created `dist/` folder to your static hosting like Surge, GitHub Pages or nginx.
```

Comment:
I wouldn't leave NGINX here without explanation as it's a reverse proxy / web server and can be used for hosting both, static sites and dynamic SSR.